src/ui/map.js

```diff
@@ -99,6 +101,11 @@ type MapOptions = {
 
 const defaultMinZoom = 0;
 const defaultMaxZoom = 22;
+
+// the default values, but also the valid range
+const defaultMinPitch = 0;
+const defaultMaxPitch = 60;
```

Comment:
Instead of defining both the default and limits, since the defaults are going to be the limits, I've used this constant for both the default and the limits.